include/libswiftnav/signal.h

```diff
@@ -22,24 +22,30 @@
 /* Number of satellites in each constellation. */
 #define NUM_SATS_GPS 32
 #define NUM_SATS_SBAS 19
-#define NUM_SATS (NUM_SATS_GPS + NUM_SATS_SBAS)
+#define NUM_SATS_GLO 24 //NOTE: DT: check this
```

Comment:
This should be correct. Current GLONASS ICD only supports up to 24 satellites. However there are plans to expand the GLONASS almanac to support more satellites in the future.